numpyro/infer/renyi_elbo.py

```diff
@@ -0,0 +1,80 @@
+from jax import random, vmap
+from jax.lax import stop_gradient
+from jax.scipy.special import logsumexp
+import jax.numpy as np
+from numpyro.infer.elbo import ELBO
+from numpyro.infer.util import log_density
+from numpyro.handlers import replay, seed
+
+
+class RenyiELBO(ELBO):
+    r"""
+    An implementation of Renyi's :math:`\alpha`-divergence variational inference following reference [1]. 
+    In order for the objective to be a strict lower bound, we require
+    :math:`\alpha \ge 0`. Note, however, that according to reference [1], depending
+    on the dataset :math:`\alpha < 0` might give better results. In the special case
+    :math:`\alpha = 0`, the objective function is that of the important weighted
+    autoencoder derived in reference [2].
+    
+    **Note:** Setting :math:`\alpha < 1` gives a better bound than the usual ELBO.
```

Comment:
nit: probably you want to use `.. note:: Setting ...` with 4-space indentations for the following lines?

Reply:
@TuanNguyen27 How about following my above comment?

Reply:
Hmm...locally i had what you commented but somehow it's not reflected in the PR, I'll try to update it now. Sorry for the confusion!